### ISSUE TITLE
KC3Kai won't run, possibly due to flash player or chrome

### ISSUE THREAD

**Issue by ltarmyfox** [KC3Kai/KC3Kai#1978]
****Chrome Version - 57.0.2987.133,  OS Windows 10,   64-bit,   KC3 Kai 31.4.0 from Chrome Web store**

**Steps to Reproduce- Unknown, just doesn't start or starts up to main screen then lags to crash. Worked last afternoon, 4-15-17, and started lagging towards the night. Possible it's because of flash player or google messing with chrome again.**

I've already done the obvious clear cache, uninstall/reinstall of KC3. Usually it would fix it, but something else is causing it to not work is my guess. This might be related to issue #1973 since he/she is suffering similar problems though the error i got were different.

Screen Shot (if applicable):  
![image](https://cloud.githubusercontent.com/assets/27599075/25068875/a383c466-223e-11e7-937c-0dbf8b276a61.png)

Error Report (if applicable): What is different from issue #1973 is that i didn't get any errors, ignoring the connection timed out ones. And i have absolutely no idea of the *uncaught* i'm seeing.

P.s I thought it might be flash player and tried to use Chrome://plugins ,but either Goggle temporarily or permanently remove that command meaning I have no way (that I know of) of checking on flash player. If someone knows another way to checking Flash Player; please tell me if there is one. Thank you.

**Comment by FOG-Yamato** [comment]
Sounds like there's a unstable connection at hand here, all those error are not KC3 related, ERR_CONNECTION_TIMED_OUT comes up when there is no response from the server in a set interval of time.

**Comment by ltarmyfox** [comment]
I use to run KC3 no problem, so I can't explain the connection problems. Normally, I start up an extension called kancolle launcher to get the API and then have KC3 take over, but i can't get an API to appear. It's not KC3 problem if i can't get an api. But, i genuinely do not know what I can possible do fix this.

**Comment by ltarmyfox** [comment]
And things get weirder for me. I'm using my backup laptop to see if it was my router or internet connection like you thought. It does work on the backup, granted it took several tries, but i got so many errors and i don't understand the difference between my two laptops.

Here's a screenshot of the errors:

![image](https://cloud.githubusercontent.com/assets/27599075/25078738/fa4a9a20-2303-11e7-977d-f4ec24806dee.png)


I just don't get it. I can't read the code like everyone else here: i'm just a guy using a program a group of people made.  

Also, on my main laptop, i thought that maybe it was chrome uninstalled/reinstalled it and ran into the problem I knew was going to be there. There is no way to turn adobe flash player on since Google has removed chrome://plugins and have no idea when/if it will come back. 

This isn't a problem just for me ,but possibility for some others users as well. I hope someone looks into this problem and hopefully figures out a solution.

**Comment by mikenzb** [comment]
Have you tried turning off your adblocker? 

> Normally, I start up an extension called kancolle launcher to get the API and then have KC3 take over

Any reason why you don't just use the DMM mode with KC3 customizations and let KC3 take over from there? 

Looking at the webstore page for kancolle launcher it does what KC3 should already do, if you're having issues getting past the region block in KC3, make sure you have force cookie hack enabled in the KC3 settings and make sure your DMM site is in Japanese.

**Comment by FOG-Yamato** [comment]
all those errors are from the ads not being able to reach their servers because of the adblock. KC3 should work without any help from a 3rd party plugin

**Comment by ltarmyfox** [comment]
So, all adblocks are disabled for dmm and that helped with some of the errors, but still having problems.

**Status of main laptop**: I still can't run KC3 at all (no idea if flash player is set to always allow is on and i can't check)

**Status of backup laptop**: It does run ,although there are some new errors (I think they are errors) that appear. Someone can tell me if these are a problem. 

![image](https://cloud.githubusercontent.com/assets/27599075/25094532/968886f2-2365-11e7-9f50-122ea90264c6.png)

Can someone also explain what is the Uncaught i'm seeing there?

**Comment by FOG-Yamato** [comment]
Don't worry about those, you tried how we told you to use cookie hack and have DMM in Japanese and only use KC3?

**Comment by ltarmyfox** [comment]
The advice helped for my backup laptop.

My main laptop still will not run KC3. Yes, cookie hack is on and Yes, DMM is in full Japanese, while using only KC3. 

![image](https://cloud.githubusercontent.com/assets/27599075/25109630/0acabc56-23ac-11e7-8cc5-c01446d5eb1a.png)

I just don't understand whats wrong.

**Comment by ethrundr** [comment]
Have you tried to reinstall/update your flash player?
Indeed, Google has their own built-in player, but sometimes, reinstall a new one is necessary to fix bugs

**Comment by ltarmyfox** [comment]
I just did uninstall/reinstall. Still no good.

Although a new error popped up that I haven't seen in my several attempts (error 404). No idea if its a common error or not, so someone needs to tell me. Or has any more ideas.

![image](https://cloud.githubusercontent.com/assets/27599075/25417431/0f2ba780-2a12-11e7-94dd-090a20dbf126.png)

**Comment by FOG-Yamato** [comment]
That 404 is not relevant, seems like the browser can't reach the game server, check your internet connection, make sure it's stable and does not have locked addesses

**Comment by ltarmyfox** [comment]
The internet connection is stable, i checked over and over, but a locked address?

**Comment by FOG-Yamato** [comment]
yeah, your ISP might block some addreses for different reasons
try a VPN and see if it loads

**Comment by ltarmyfox** [comment]
its shouldn't be locked if I can still use KC3 on my backup laptop. Its be a week since this started and not once has the backup had any problems.

**Comment by FOG-Yamato** [comment]
Correct, I would say to check for anything that can block the connection on your computer, Firewall, anti virus, other programs. That ERR_CONNECTION_TIMED_OUT means that the connection was made but no response came back and it got terminated (a.k.a it timed out).

**Comment by ltarmyfox** [comment]
I know for fact that it isn't the firewall or the anti-virus. What else could cause it? Any ideas what else I should be looking for?

**Comment by FOG-Yamato** [comment]
anything that can block your connection, could be anything that is network related.
